utils/client_helper.go

```diff
@@ -334,6 +336,67 @@ func HasFIPS(ctx context.Context, client runtimeclient.Client) (bool, error) {
 	return machineConfig.Spec.FIPS, nil
 }
 
+func GetAdditionalTrustBundleFromConfigmap(ctx context.Context, client client.Client, configmapName string) (string, error) {
```

Comment:
`configmapName` not used within this func, any reason to maintain it?

Reply:
It should have been use, it not being used is a bug. Will fix